.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: QMlzuOK6VEAQhMdQzJOhVSFpnUwjCKHGh
```

Comment:
Looks like we won't even need it: https://github.com/lemurheavy/coveralls-public/issues/208#issuecomment-34496827 :smile: